--- a/includes/menu.inc
+++ b/includes/menu.inc
@@ -19,8 +19,8 @@
 function _civictheme_preprocess_menu_items(array &$items, bool $expand_all = FALSE): void {
   foreach ($items as &$item) {
     $item += [
-      'is_new_window' => $item['url'] instanceof Url ? $item['url']->isExternal() : FALSE,
-      'is_external' => $item['url'] instanceof Url ? $item['url']->isExternal() : FALSE,
+      'is_new_window' => isset($item['url']) && $item['url'] instanceof Url ? $item['url']->isExternal() : FALSE,
+      'is_external' => isset($item['url']) && $item['url'] instanceof Url ? $item['url']->isExternal() : FALSE,
     ];
 
     // Set as expanded based on $expand_all or item's own value or FALSE.
